.github/workflows/tutorial-support.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: tutorial-support
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the devel branch
+  push:
+    branches: [ master, devel ]
+  pull_request:
+    branches: [ master, devel ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: checkout latest GpsApp repo
+        uses: actions/checkout@v2
+        with:
+          repository: r9-pena/gps-tutorial
```

Comment:
Probably need to use `fprime-community/gps-tutorial` when this is ready to start CI-ing.

Reply:
I've been testing it my personal fork. It has been successfully running. I think it should work the same once I update the repository for the gps-tutorial.